src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_TicketHistoryScreen.tsx

```diff
@@ -32,21 +33,24 @@ export const Profile_TicketHistoryScreen: React.FC = () => {
         <ScreenHeading ref={focusRef} text={t(TicketHistoryTexts.header)} />
       )}
     >
-      <FareContractAndReservationsList
-        fareContracts={expiredFareContracts}
-        reservations={rejectedReservations}
+      <ScrollContainer
         isRefreshing={isRefreshingFareContracts}
         refresh={resubscribeFirestoreListeners}
-        now={serverNow}
-        emptyStateTitleText={t(
-          TicketingTexts.activeFareProductsAndReservationsTab
-            .emptyTicketHistoryTitle,
-        )}
-        emptyStateDetailsText={t(
-          TicketingTexts.activeFareProductsAndReservationsTab
-            .emptyTicketHistoryDetails,
-        )}
-      />
+      >
+        <FareContractAndReservationsList
```

Comment:
I think we can just start using the new screen component here as well, in the same PR where it is added.

Reply:
will add on correct PR